### PR TITLE
Fixed initialization of IPythonKernelCallback

### PR DESCRIPTION
**Pull request by philippjfr**
As the title suggests this PR ensures the plot is initialized correctly when using the IPythonKernelCallback.

### CODE REVIEW

File: datashader/callbacks.py [hunk 1]
```diff
@@ -62,7 +62,8 @@ def __init__(self, bokeh_plot, callback, throttle=100, **kwargs):
         w, h = self.p.plot_width, self.p.plot_height
         xmin, xmax = self.p.x_range.start, self.p.x_range.end
         ymin, ymax = self.p.y_range.start, self.p.y_range.end
-        callback((xmin, xmax), (ymin, ymax), w, h, **kwargs)
+        self.update_image({'xmin': xmin, 'xmax': xmax, 'ymin': ymin,
+                           'ymax': ymax, 'w': w, 'h': h})
```
Comment (jbednar) [inline datashader/callbacks.py]: Was this meant to be `redraw_image`?  There's no `update_image` anymore.

Comment (philippjfr) [inline datashader/callbacks.py]: Yes it is, thanks for catching that.

Comment (philippjfr) [inline datashader/callbacks.py]: Fixed.

**Comment by jbednar** [comment]
Thanks so much!  It seems to work fine now, which we couldn't manage ourselves!

**Pull request merged**